downstream/assemblies/platform/assembly-migrate-legacy-venv-to-ee.adoc

```diff
@@ -14,20 +14,20 @@ ifdef::context[:parent-context: {context}]
 
 [role="_abstract"]
 
-If upgrading from older versions of {ControllerName} to 4.0 or later, the controller has the ability to detect previous versions of virtual environments associated with Organizations, Inventory, and Job Templates; and inform you that you will need to migrate to the new {ExecEnvName} model. A brand new installation of {ControllerName} creates two virtualenvs during installation--one is used to run the controller itself, while the other is used to run Ansible. Like legacy virtual environments, {ExecEnvName} allow the controller to run in a stable environment, while allowing you to add or update modules to your {ExecEnvName} as necessary to run your playbooks.
+include::platform/con-why-ee.adoc[leveloffset=+1]
+
+== About migrating legacy venvs to {ExecEnvName}
 
-.Migrate legacy venvs to {ExecEnvName}
+If upgrading from older versions of {ControllerName} to 4.0 or later, the controller has the ability to detect previous versions of virtual environments associated with Organizations, Inventory, and Job Templates; and inform you that you will need to migrate to the new {ExecEnvName} model. A brand new installation of {ControllerName} creates two virtualenvs during installation--one is used to run the controller itself, while the other is used to run Ansible. Like legacy virtual environments, {ExecEnvName} allow the controller to run in a stable environment, while allowing you to add or update modules to your {ExecEnvName} as necessary to run your playbooks.
```

Comment:
Suggest rewording for active voice:
When upgrading from older versions of {ControllerName} to version 4.0 or later, the controller can detect previous versions of virtual environments associated with Organizations, Inventory and Job Templates and informs you to migrate to the new {ExecEnvName} model.